src/Parsers/Access/ParserCreateQuotaQuery.cpp

```diff
@@ -107,23 +107,182 @@ namespace
         });
     }
 
+    template <typename T, typename = std::enable_if_t<std::is_same_v<T, QuotaValue> || std::is_same_v<T, double>>>
+    bool multipleBinarySuffix(T & max_value, int power_of_two)
```

Comment:
It's strange that we have suffix-parsing code here while we already support it for Settings.

Reply:
ReadHelpers.h, `parseWithSizeSuffix`